xrviz/dashboard.py

```diff
@@ -20,23 +20,28 @@
 class Dashboard(SigSlot):
     """
     This section provides access to the complete generated dashboard,
```

Comment:
Since this is object-oriented programming, this **is** the dashboard. It is not a section, and it does not provide access, it simply is.

"This is the main entry point to XrViz, and generates an interactive GUI for a given dataset"